src/mprismetadata.cpp

```diff
@@ -19,6 +17,212 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+/*!
+    \qmltype MprisMetaData
+    \inqmlmodule Amber.Mpris
+    \brief Provides media metadata information
+
+    Provides access to information about media content.
+
+    All properties may be undefined if not applicable or known.
+    A default trackId will be returned if one is not defined.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::trackId
+    \brief Identifies the track inside a playlist
+
+    Identifier for the current track, should be a valid DBus object path.
```

Comment:
I could read that as the application would be even responsible of registering a dbus object. The actual MPRIS API looks more relaxed: "Note that this should be a valid D-Bus object id, although clients should not assume that any object is actually exported with any interfaces at that path. " 

-> Could adjust a bit. ~"...a valid DBus object path format. Doesn't need anything registered on the path."

Reply:
Better?

Reply:
Looks like you hit the source of my earlier confusion today: there's a same named branch on the repository but the PR is from your clone :)

Reply:
Ouch. should change origin to something else, so I'd not do write it "automatically".

Reply:
Removed the rogue branch, although it doesn't help much.